practice/bio/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+
+<html lang= "en">
+
+  <head>
+    <meta charset="utf-8">
+    <title>Bio</title>
+    <link rel="stylesheet" href="site.css" />
+  </head>
+
+  <body>
+    <header>
+      <h1>John Cena Bio</h1>
+    </header>
+    <nav>
+    </nav>
```

Comment:
You're not required to have a `nav` element and sometimes having empty elements can make layout tricky.